iothub_client/inc/iothub_message.h

```diff
@@ -191,9 +191,12 @@ MOCKABLE_FUNCTION(, MAP_HANDLE, IoTHubMessage_Properties, IOTHUB_MESSAGE_HANDLE,
 *
 * @param   iotHubMessageHandle Handle to the message.
 *
-* @param   key name of the property to set.  Note that when sending messages via the HTTP transport, this value must not contain spaces.
+* @param   key name of the property to set. Note that when sending messages via the HTTP transport, this value must not contain spaces.
 *
-* @param   value of the property to set.
+* @param   value of the property to set. Note that when sending messages via the HTTP transport, this value must not contain spaces.
+*
+*            @b NOTE: Property names and values must not contain spaces and can only contain ASCII alphanumeric characters, 
+*            plus {'!', '#', '$', '%, '&', ''', '*', '+', '-', '.', '^', '_', '`', '|', '~'}.
```

Comment:
I think mollie is trying to validate this list.

Reply:
Working on this.  Please wait to merge this, as we may need to change what is written here.  Thanks.